core-infrastructure/src/db/Core.Database/Scripts/089-CreateSchoolCensusHistoricWithNullsView.sql

```diff
@@ -0,0 +1,60 @@
+DROP VIEW IF EXISTS SchoolCensusHistoricWithNulls
+GO
+
+CREATE VIEW SchoolCensusHistoricWithNulls AS
+  SELECT URN
+       , Year
+       , FinanceType
+       , OverallPhase
+       , TotalPupils
+       , CASE
+             WHEN WorkforceFTE IS NULL OR WorkforceFTE = 0 THEN NULL
```

Comment:
Note: I'm adding this to specifically avoid divide-by-zero errors, hence the `= 0` rather than a `<= 0`.